### ISSUE TITLE
Tooltip is missing when using keyboard to highlight tree items.

### ISSUE THREAD

**Issue by smartguest** [microsoft/vscode#141021]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: N/A
- OS Version: Windows 10

## Context: 

This issue was originally reported in vscode-mssql:
https://github.com/microsoft/vscode-mssql/issues/17201

## Steps to Reproduce:

1. Select a treeitem using tab and arrow keys,

## Actual: 

1. When using arrow keys to move to a tree item in a window, tooltip is not available with the keyboard keys.

## Expected:

When using arrow keys to move to data in a window, tooltip should be available with the keyboard keys which is available with mouse.

**Comment by alexr00** [comment]
@isidorn is there a usual way to access a tooltip via keyboard that we should be following?

**Comment by isidorn** [comment]
@alexr00 yeah, `cmd + k, cmd + i`. Works for all editor hovers. Make sure that when user access the tooltip like this that the focus is moved inside the tooltip - so the screen reader would read the proper thing

**Comment by alexr00** [comment]
I have added the `ctrl + k, ctrl + i` keyboard shortcut for extension tree hovers. However, when the hover is escaped, focus does not return to the tree as I think is correct. @Tyriar, is there an option I am missing when using `focus = true` for ensuring that focus is returned to the previously focused element when the hover is escaped?

**Comment by isidorn** [comment]
@alexr00 great, thanks! Looking at the code it does not look like focus is passed to the hover (I might be misinterpreting). Is it possible to move the focus to the custom hover when this command is triggered by keybinding?

**Comment by alexr00** [comment]
Focus is passed to the hover. I confirmed that NVDA reads the hover when shown via the keybinding.

**Comment by isidorn** [comment]
Thanks! I missed that `true`.
